src/main/java/de/zalando/ep/zalenium/container/kubernetes/KubernetesContainerClient.java

```diff
@@ -1,16 +1,24 @@
 package de.zalando.ep.zalenium.container.kubernetes;
 
+import de.zalando.ep.zalenium.container.ContainerClient;
```

Comment:
Please revert all changes to this file. Also kubernetes already has its own incompatible notion of cpu limits and requests.

Reply:
I agree, this file does not need to be touched by this PR.